runtime/integration-tests/src/mock.rs

```diff
@@ -226,27 +408,31 @@ impl Builder<PendulumCurrencyId> for ExtBuilderParachain<PendulumCurrencyId> {
 	}
 
 	fn build(self) -> TestExternalities {
+		let assets_metadata_pendulum = assets_metadata_for_registry_pendulum();
+		let assets_metadata_amplitude = assets_metadata_for_registry_amplitude();
 		match self.chain {
 			ParachainType::Pendulum => {
 				use pendulum_runtime::{Runtime, System};
-				build_parachain_with_orml!(
+				build_parachain_with_orml_and_asset_registry!(
 					self,
 					Runtime,
 					System,
 					NATIVE_INITIAL_BALANCE,
 					ORML_INITIAL_BALANCE,
-					PendulumCurrencyId
+					PendulumCurrencyId,
+					assets_metadata_pendulum
 				)
 			},
 			ParachainType::Amplitude => {
 				use amplitude_runtime::{Runtime, System};
-				build_parachain_with_orml!(
+				build_parachain_with_orml_and_asset_registry!(
 					self,
 					Runtime,
 					System,
 					NATIVE_INITIAL_BALANCE,
 					ORML_INITIAL_BALANCE,
-					PendulumCurrencyId
+					PendulumCurrencyId,
```

Comment:
This issue existed before but I assume it should be (?)
```suggestion
					AmplitudeCurrencyId,
```

Reply:
Yes, although they are the same type, it makes sense to maintain naming convention.